src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.command.Command;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * Represents the main functions of Duke chatbot and initializes require variables.
+ */
+public class Duke {
+
+    private List<Task> list = new ArrayList<>();
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+    private boolean isExit;
+
+    /**
+     * Initialises Duke chatbot.
+     * @param filePath directory of where tasks are saved.
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+            System.out.println("list size-" + tasks.listSize());
```

Comment:
Maybe can remove this console log if it's for debugging purposes.

Reply:
Oh i forgot about that, thanks!